ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -388,27 +388,27 @@
 
 | Interface | Ethernet Segment Identifier | Multihoming Redundancy Mode | Route Target |
 | --------- | --------------------------- | --------------------------- | ------------ |
-{%         for evpn_es_ethernet_interface in evpn_es_ethernet_interfaces | arista.avd.natural_sort %}
-{%             set esi = ethernet_interfaces[evpn_es_ethernet_interface].evpn_ethernet_segment.identifier | arista.avd.default("-") %}
-{%             set redundancy = ethernet_interfaces[evpn_es_ethernet_interface].evpn_ethernet_segment.redundancy | arista.avd.default("all-active") %}
-{%             set rt = ethernet_interfaces[evpn_es_ethernet_interface].evpn_ethernet_segment.route_target | arista.avd.default("-") %}
-| {{ evpn_es_ethernet_interface }} | {{ esi }} | {{ redundancy }} | {{ rt }} |
+{%         for evpn_es_ethernet_interface in evpn_es_ethernet_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
the conversion and sorting is not needed here, since you build the `evpn_es_ethernet_interfaces` list above in line 369-382. With these changes the `evpn_es_ethernet_interfaces` goes from being a list of keys to a list of dicts. Same for `evpn_dfe_ethernet_interfaces` and `evpn_mpls_ethernet_interfaces` below